ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/input-variables.md

```diff
@@ -841,6 +841,14 @@ roles/eos_cli_config_gen/docs/tables/static-routes.md
 roles/eos_cli_config_gen/docs/tables/vrfs.md
 --8<--
 
+## Security
+
+### IP Security
```

Comment:
Please move macsec to this new section. Maybe call it dataplane security since we already have endpoint security.

Reply:
Macsec is already under endpoint security - should I move it?

Reply:
I would say so yes and rename this one to `Data Plane Security`